src/utils/__tests__/version.test.ts

```diff
@@ -110,6 +114,15 @@ describe('parseVersion', () => {
     });
   });
 
+  test('parses a Python-style post release version', () => {
+    expect(parseVersion('1.2.3-1')).toEqual({
+      major: 1,
+      minor: 2,
+      patch: 3,
+      pre: '1',
```

Comment:
might be worth a comment that it misinterprets this -- but it's ok as long as we don't utilize this function for python in the same way